validate_yamls.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+
+import argparse
+import glob
+import os
+import sys
+
+from lib import exception
+from lib.utils import is_package_installed
+from lib.utils import run_command
+
+def validate_package_metadata(package_metadata_file_path):
+
+    try:
+        run_command("yamllint %s" % package_metadata_file_path)
+    except exception.SubprocessError as e:
+        #pylint: disable=no-member
+        print("validation of package metadata file %s failed, output: %s" % (package_metadata_file_path, e.stdout))
+        return
+
+def validate_packages_metadata(base_dir):
+    """
+    Validate specification files of rpm packages built by Open Power Host OS
+    """
+
+    packages = os.listdir(base_dir)
+
+    for package in packages:
+        files = glob.glob("{base_dir}/{package_name}/{package_name}.yaml".format(
```

Comment:
Same here.  Since `base_dir` is given, code can walk through this directory recursively and search for all `*.yaml` files to validate.